os/arch/Kconfig.chip

```diff
@@ -735,6 +735,27 @@ config RAM_SIZE
 		does not execute out of RAM but from FLASH, then you may designate
 		any block of RAM as "primary."
 
+config RAMx_START
+	string "List of start address for additional heap region"
+	default "0x00000000,"
+	depends on MM_REGIONS != 1
+	---help---
+		The start address list of the additional heap regions.
+		When you want to use separated heap region in a physical RAM or
+		two different physical RAMs, set MM_REGIONS first at memory
+		management, and then set RAMx_START and RAMx_SIZE to support
+		several heap regions.
+		Commas show each region.
+
```

Comment:
Give an example of region definition. e.g. {0x20000000, 0x40000000}. This will give better picture and overall purpose of how it can be used. Same for SIZE.  "Commas show each region." This is confusing, Instead mention "Each region is separated by ',' as shown in example."

Reply:
I will. Thank you.